doc/introduction/interfaces/jax.rst

```diff
@@ -0,0 +1,148 @@
+.. _jax_interf:
+
+JAX interface
+=================
+
+Born out of the autograd package, `JAX <https://jax.readthedocs.io/en/latest/index.html>`_ is the next generation of differentiable functional computation, 

```

Comment:
Lol, is that a politically controversial statement? :)

Reply:
Maybe "next iteration" instead of generation?